GiniVision/Classes/ImageMetaInformationManager.swift

```diff
@@ -71,8 +71,20 @@ internal class ImageMetaInformationManager {
         }
     }
     
-    func imageData(withCompression compression: CGFloat = JPEGDefaultCompression) -> Data? {
-        return generateImage(withMetaInformation: metaInformation, andCompression: compression)
+    // Returns the current image but with all meta information added
+    func addMetaInformationToImage(withCompression compression: CGFloat = JPEGDefaultCompression) -> Data? {
```

Comment:
This name is misleading. it makes me think it will add the meta info to the existing image, but it actually returns a new image data. A more swift-like name would be `imageByAddingMetadata` or something.

Reply:
Yes, the name you suggest is more swift-like. 👍